packages/jbrowse-web/src/sessionModelFactory.ts

```diff
@@ -395,27 +395,27 @@ export default function sessionModelFactory(pluginManager: any) {
       },
 
       addSavedSession(sessionSnapshot: SnapshotIn<typeof self>) {
-        return getParent(self).jbrowse.addSavedSession(sessionSnapshot)
+        return getParent<any>(self).jbrowse.addSavedSession(sessionSnapshot)
       },
 
       removeSavedSession(sessionSnapshot: any) {
-        return getParent(self).jbrowse.removeSavedSession(sessionSnapshot)
+        return getParent<any>(self).jbrowse.removeSavedSession(sessionSnapshot)
       },
 
       renameCurrentSession(sessionName: string) {
-        return getParent(self).renameCurrentSession(sessionName)
+        return getParent<any>(self).renameCurrentSession(sessionName)
```

Comment:
what are all these `<any>` template args about?

Reply:
getParent in mobx-state-tree is a templated return type. previously it had return type any. we can narrow it down to a more specific thing if there is interest in typing the rootmodel, etc.

Reply:
that is to say, it was just changed in typescript in 3.15.0 to be unknown return type